src/types/auth.ts

```
@@ -0,0 +1,30 @@
+
+export interface LoginRequest {
+  email: string;
+  password: string;
+}
+
+export interface TokenResponse {
+  accessToken: string;
+  refreshToken: string;
+}
+
+export interface AuthenticatedRequest<Params = {}, Query = ParsedQs, Body = {}> extends Request<Params, any, Body, Query> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)